packages/webapp/src/pages/induction/[id].tsx

```diff
@@ -1,128 +1,106 @@
+    const status = getInductionStatus(induction, endorsements);
 
-    const renderInductionStep = useMemo(() => {
+    const renderInductionJourney = useMemo(() => {
         if (!induction) return "";
-
-        if (reviewStep === "profile") {
-            return (
-                <InductionStepProfile
-                    induction={induction}
-                    isCommunityActive={isCommunityActive}
-                    isEndorser={isEndorser}
-                    isReviewing
-                />
-            );
-        }
-
-        if (reviewStep === "video") {
-            return (
-                <InductionStepVideo
-                    induction={induction}
-                    isEndorser={isEndorser}
-                    isReviewing
-                />
-            );
-        }
-
-        switch (status) {
-            case InductionStatus.waitingForProfile:
+        switch (userRole) {
+            case InductionRole.Inviter:
+            case InductionRole.Endorser:
                 return (
-                    <InductionStepProfile
+                    <InviterWitnessJourney
+                        endorsements={endorsements}
                         induction={induction}
-                        isEndorser={isEndorser}
-                        isCommunityActive={isCommunityActive}
+                        inductionStatus={status}
                     />
                 );
-            case InductionStatus.waitingForVideo:
+            case InductionRole.Invitee:
                 return (
-                    <InductionStepVideo
+                    <InviteeJourney
+                        endorsements={endorsements}
                         induction={induction}
-                        isEndorser={isEndorser}
+                        inductionStatus={status}
                     />
                 );
-            case InductionStatus.waitingForEndorsement:
+            case InductionRole.Member:
+            case InductionRole.Unknown:
+            case InductionRole.Unauthenticated:
                 return (
-                    <InductionStepEndorsement
-                        induction={induction}
+                    <ThirdPartyJourney
                         endorsements={endorsements}
-                        isCommunityActive={isCommunityActive}
-                        setReviewStep={setReviewStep}
+                        induction={induction}
+                        inductionStatus={status}
                     />
                 );
-            default:
-                return "";
         }
-    }, [induction, isCommunityActive, isEndorser, endorsements, reviewStep]);
+    }, [induction, endorsements, status, userRole]);
+
+    // TODO: We need a common loading spinner component.
+    if (isLoading) {
+        return (
+            <SingleColLayout title="Loading">
+                <Card title="Loading...">...</Card>
+            </SingleColLayout>
+        );
+    }
```

Comment:
yeah, it's a good idea to get a common loading spinner... aside from that I see that here we are replacing the whole layout. can't we just have the same wrapper with the `Induction #` title?

Reply:
Yeah, I don't see why not. I'll modify. 👍